exercises/run-length-encoding/src/example/java/RunLengthEncoding.java

```diff
@@ -0,0 +1,47 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RunLengthEncoding {
+    public String encode(String data) {
+        String encodedData = "";
+
+        for (int i = 0; i < data.length(); i++) {
+            char c = data.charAt(i);
+
+            if (i != 0 && c == data.charAt(i - 1)) continue;
```

Comment:
Good idea.  I try to keep the body of code on the same line as the header if it's only one line and doesn't require curly braces.  This way, if I add more code later, it's less likely for me to have problems because I will see that there are no curly braces.  I suppose though, that it's better just to insert the curly braces anyways.  I think that it's also more common to see it that way.